src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -144,16 +146,28 @@ public XpathQueryGenerator(DetailAST rootAst, int lineNumber, int columnNumber,
     }
 
     /**
-     * Returns child {@code DetailAst} element of the given root,
-     * which has child element with token type equals to {@link TokenTypes#IDENT}.
+     * Returns child {@code DetailAst} element of the given root, which has value attribute.
```

Comment:
Remember to update javadocs too. This still says value attribute.

Reply:
done